trame/app/__init__.py

```diff
@@ -1,29 +1,38 @@
+from __future__ import annotations
+
 import logging
-from trame_server import Server, Client
-from trame_server.core import set_default_client_type
+from typing import Literal
+
 from trame_client.widgets.core import VirtualNode
+from trame_server import Client, Server
+from trame_server.core import set_default_client_type
 
 # Ensure this is imported so that mimetypes.init() is decorated
 import trame.app.mimetypes  # noqa: F401
 
 DEFAULT_NAME = "trame"
-AVAILABLE_SERVERS = {}
-AVAILABLE_CLIENTS = {}
+AVAILABLE_SERVERS: dict[str, Server] = {}
+AVAILABLE_CLIENTS: dict[str, Client] = {}
 
 logger = logging.getLogger(__name__)
 
 set_default_client_type("vue3")
 
 
-def apply_client_type(server, client_type=None):
+def apply_client_type(server: Server, client_type: str | None = None) -> Server:
```

Comment:
Maybe `client_type: Literal["vue2", "vue3"]` instead of `str`?

Reply:
yes I guess that will be more precise.

Reply:
Sorry about this late change, should I open a new PR?

Reply:
Yes, but are you planning for other API typing? If so, you may want to bundle that with some more changes...